index_test.go

```diff
@@ -91,6 +94,29 @@ func TestDocSectionInvalid(t *testing.T) {
 	}
 }
 
+func TestBloomSkip(t *testing.T) {
+	for _, tc := range []struct {
+		skip bool
+		want int
+	}{
+		{false, 1},
+		{true, 0},
+	} {
+		if tc.skip {
+			os.Setenv("ZOEKT_DISABLE_BLOOM", "1")
```

Comment:
FYI there is `testing.T.Setenv` since go1.17. Our CI for zoekt is not on that yet, so nothing to change in this PR.